packages/fast-css-editor-react/src/position/position.props.ts

```diff
@@ -4,6 +4,7 @@ import { CSSPositionClassNameContract } from "./position.style";
 export enum PositionValue {
     static = "static",
     absolute = "absolute",
+    relative = "relative",
```

Comment:
should we just add all of them?

Reply:
Chatted offline. Agreed to add `relative` and `fixed`